app/views/steps/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<main class="govuk-main-wrapper">
```

Comment:
Again this is creating a second `main` element (mentioned in review of preceding PR)

Reply:
I think this is [resolved in the earlier PR](https://github.com/dxw/dfsseta-apply-for-landing-ruby/blob/b4ef9fc1bc152c171af3bb9d304b7395aaa1b6fa/app/views/layouts/application.html.erb#L54) in which the outer layout `govuk-width-container` is now a `<div>` element rather than `<main>`